.contributing/the-process.md

```diff
@@ -61,27 +61,24 @@ For each use-case there will be three types of digital goodness:
 
     -   Sequence Diagram
 
-Create: 
+Inside Google Drive Folder "x. *Section*: Sections Planning (CL)", create:
```

Comment:
i split up asana, github, and google drive tasks from "Create: " as it gave the freedom to give context with the title, and looked alot clearer after i did it

Reply:
🙏